src/assets/javascript/main.js

```diff
@@ -1,15 +1,14 @@
 import React from "react";
 import ReactDOM from "react-dom";
 import Calculator from "../../components/Calculator";
+import CalculatorProcess from "../../services/CalculatorProcess";
+import VisorInteractor from "../../services/VisorInteractor";
 import "../../assets/css/custom.scss";
 
-const calculatorProcess = require("../../services/CalculatorProcess");
-const visorInteractor = require("../../services/VisorInteractor");
-
 ReactDOM.render(
 
 	<Calculator id="calculator" />,document.querySelector("body")
 
 ); 
 document.querySelectorAll("#calculator button")
-	.forEach(button => button.addEventListener("click", () => calculatorProcess(button.dataset.value,visorInteractor) ));
+	.forEach(button => button.addEventListener("click", () => CalculatorProcess(button.dataset.value,VisorInteractor) ));
```

Comment:
@eerison 
should i change these two functions to components or react components it's better just for generate html ans stuffs ?
i mean, i was spend a lot of time trying to change these two function `CalculatorProcess()` and `VisorInteractor()` to components i didn't even know if it was necessary

Reply:
done